totalRP3/Modules/Register/Characters/RegisterCharacteristics.lua

```diff
@@ -1119,6 +1119,15 @@ function setEditDisplay()
 		frame.miscIndex = frameIndex;
 		_G[frame:GetName() .. "Icon"].IC = miscStructure.IC or TRP3_InterfaceIcons.Default;
 		_G[frame:GetName() .. "NameField"]:SetText(miscStructure.NA or loc.CM_NAME);
+		-- Disabling name edition on presets
+		if miscStructure.ID == TRP3_API.MiscInfoType.Custom then
+			_G[frame:GetName() .. "NameField"]:Show();
+			_G[frame:GetName() .. "PresetNameText"]:Hide();
+		else
+			_G[frame:GetName() .. "NameField"]:Hide();
+			_G[frame:GetName() .. "PresetNameText"]:Show();
+			_G[frame:GetName() .. "PresetNameText"]:SetText(miscStructure.NA or loc.CM_NAME);
```

Comment:
```suggestion
			frame.PresetName:Hide();
		else
			_G[frame:GetName() .. "NameField"]:Hide();
			frame.PresetName:Show();
			frame.PresetName:SetText(miscStructure.NA or loc.CM_NAME);
```

I imagine the use of global names here was more for consistency reasons, but I think it just makes sense to use parent keys going forward where possible for newer stuff even if it makes for some ugliness like this.